test-cases/features/recovery_10min.yaml

```diff
@@ -0,0 +1,20 @@
+# TODO: need to qualify
```

Comment:
change the name of the yaml, I don't want it to mention recovery.
Also, no need the 10min.